1.7/administration/installing/cloud/aws/basic/index.md

```diff
@@ -0,0 +1,92 @@
+---
+post_title: Basic DC/OS Installation on AWS
```

Comment:
@pyronicide There's now a landing page that gives people 2 options: basic or advanced. If we make basic the default path, I think we're doing a disservice to our users (not giving them an option). What exactly is your concern?